src/components/ConnectionLayout.tsx

```diff
@@ -60,6 +60,12 @@ type ConnectionLayoutProps = {
 
     /** Name of the current connection */
     connectionName: ConnectionName;
+
+    /** Block the screen when the connection is not empty */
+    reverseConnectionEmptyCheck?: boolean;
+
+    /** Handler for back button press */
+    onBackButtonPress?: () => void;
```

Comment:
@shubham1206agra May be this will conflict with your Export. I'll adjust once your PR is merged.

Reply:
Fixed this.